packages/web-react/src/components/FileUploader/FileUploaderList.tsx

```diff
@@ -6,7 +6,7 @@ import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
 import { useFileUploaderContext } from './FileUploaderContext';
 
 const FileUploaderList = (props: SpiritFileUploaderListProps) => {
-  const { label, id, attachmentComponent, inputName, ...restProps } = props;
+  const { label, id, attachmentComponent, inputName, showImagePreview, ...restProps } = props;
```

Comment:
```suggestion
  const { label, id, attachmentComponent, inputName, hasImagePreview, ...restProps } = props;
```

For boolean props, we are using `is/has` prefixes. But I know that naming is hard.